learning_rate: 0.0003
n_steps: 4096
batch_size: 64
n_epochs: 10
gamma: 0.99
gae_lambda: 0.953
clip_range: 0.2
ent_coef: 0.0
vf_coef: 0.5
max_grad_norm: 0.5
timesteps_per_stage: [50000, 100000, 150000]
